Userland/Utilities/lspci.cpp

```diff
@@ -25,12 +25,18 @@ static constexpr StringView format_numerical = "{:04x}:{:02x}:{:02x}.{} {}: {}:{
 static constexpr StringView format_textual = "{:04x}:{:02x}:{:02x}.{} {}: {} {} (rev {:02x})"sv;
 static constexpr StringView format_region = "\tBAR {}: {} region @ {:#x}"sv;
 
-static u32 read_hex_string_from_bytebuffer(ByteBuffer const& buf)
+static ErrorOr<u32> read_hex_string_from_bytebuffer(ByteBuffer const& buf)
 {
-    // FIXME: Propagate errors.
-    return AK::StringUtils::convert_to_uint_from_hex(
-        ByteString(MUST(buf.slice(2, buf.size() - 2)).bytes()))
-        .release_value();
+    auto slice_result = buf.slice(2, buf.size() - 2);
+    if (slice_result.is_error())
```

Comment:
Please use `TRY()` instead of manually expanding it.

Reply:
👍